docs/source/install/rhel6.rst

```diff
@@ -0,0 +1,291 @@
+
+    st2ctl start|stop|status|restart|restart-component|reload|clean
+
+
+-----------------
+
+At this point you have a minimal working installation, and can happily play with StackStorm:
+follow :doc:`/start` tutorial, :ref:`deploy examples <start-deploy-examples>`, explore and install packs from `st2contrib`_.
+
+But there is no joy without WebUI, no security without SSL termination, no fun without ChatOps, and no money without Enterprise edition. Read on, move on!
+
+-----------------
+
+Configure Authentication
+------------------------
+
+Reference deployment uses File Based auth provider for simplicity. Refer to :doc:`/authentication` to configure and use PAM or LDAP autentication backends. To set up authentication with File Based provider:
+
+* Create a user with a password:
+
+  .. code-block:: bash
+
+    # Install htpasswd utility if you don't have it
+    sudo yum -y install httpd-tools
+    # Create a user record in a password file.
+    sudo htpasswd -bs /etc/st2/htpasswd test Ch@ngeMe
+
+* Enable and configure auth in ``/etc/st2/st2.conf``:
+
+  .. sourcecode:: ini
+
+    [auth]
+    # ...
+    enabled = True
+    backend = flat_file
+    backend_kwargs = {"file_path": "/etc/st2/htpasswd"}
+    # ...
+
+* Restart the st2api service: ::
+
+    sudo st2ctl restart-component st2api
+
+* Authenticate, export the token for st2 CLI, and check that it works:
+
+  .. code-block:: bash
+
+    # Get an auth token and use in CLI or API
+    st2 auth test
+
+    # A shortcut to authenticate and export the token
+    export ST2_AUTH_TOKEN=$(st2 auth test -p Ch@ngeMe -t)
+
+    # Check that it works
+    st2 action list
+
+Check out :doc:`/cli` to learn convinient ways to authenticate via CLI.
+
+Install WebUI and setup SSL termination
+---------------------------------------
+`NGINX <http://nginx.org/>`_ is used to serve WebUI static files, redirect HTTP to HTTPS,
+provide SSL termination for HTTPS, and reverse-proxy st2auth and st2api API endpoints.
+To set it up: install `st2web` and `nginx`, generate certificates or place your existing
+certificates under ``/etc/ssl/st2``, and configure nginx with StackStorm's supplied
+:github_st2:`site config file st2.conf<conf/nginx/st2.conf>`.
+
+  .. code-block:: bash
+
+    # Install st2web and nginx
+    sudo yum -y install st2web nginx
+
+    # Generate self-signed certificate or place your existing certificate under /etc/ssl/st2
+    sudo mkdir -p /etc/ssl/st2
+
+    sudo openssl req -x509 -newkey rsa:2048 -keyout /etc/ssl/st2/st2.key -out /etc/ssl/st2/st2.crt \
+    -days 365 -nodes -subj "/C=US/ST=California/L=Palo Alto/O=StackStorm/OU=Information \
+    Technology/CN=$(hostname)"
+
+    # Copy and enable StackStorm's supplied config file
+    sudo cp /usr/share/doc/st2/conf/nginx/st2.conf /etc/nginx/conf.d/
+
+    # Disable default_server configuration in existing /etc/nginx/nginx.conf
+    sudo sed -i 's/default_server//g' /etc/nginx/conf.d/default.conf
+
+    sudo service nginx restart
```

Comment:
should be `nginx start` ? no point of restarting in context and extra "FAILED" on the screen.